client/src/components/map/controls/zoom.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button } from "@/components/ui/button";
+import { MinusIcon, PlusIcon } from "lucide-react";
+import { MouseEvent, useCallback } from "react";
+import { useMap } from "react-map-gl";
+
+const MapZoomControl = () => {
+  const { current: mapRef } = useMap();
+
+  const increaseZoom = useCallback(
+    (e: MouseEvent<HTMLButtonElement>) => {
+      e.stopPropagation();
+      mapRef?.zoomIn();
+    },
+    [mapRef],
+  );
+
+  const decreaseZoom = useCallback(
+    (e: MouseEvent<HTMLButtonElement>) => {
+      e.stopPropagation();
+      mapRef?.zoomOut();
+    },
+    [mapRef],
+  );
+
+  return (
+    <div className="rounded-full bg-background p-1.5 shadow-black/10 drop-shadow-md">
```

Comment:
These two buttons need a label of some sort for screen readers. I would recommend the following:
```tsx
<span className="sr-only">Zoom in</span>
```

Reply:
Those buttons also have a specific focus style: https://www.figma.com/design/HBpMppIFHR1vA0SbnUJGfO/GRASS---Design-%5BInternal%5D?node-id=529-2599&t=986E6GmpDtbfvv5L-4